pesuacademy/pages/staff.py

```diff
@@ -0,0 +1,136 @@
+import time
+# from selenium import webdriver
+# from selenium.webdriver.chrome.options import Options
+# from selenium.webdriver.common.by import By
+# from selenium.webdriver.support.ui import WebDriverWait
+# from selenium.webdriver.support import expected_conditions as EC
+from requests_html import HTMLSession
+import requests
+from bs4 import BeautifulSoup
+from ..models.staff import Staff  
+
+class StaffPageHandler:
+    @staticmethod
+    def get_staff_details() -> Staff:
+        try:
+            base_url = "https://staff.pes.edu/atoz/"
+            session=HTMLSession()
```

Comment:
Re-use the same session please. We don't want multiple sessions connected to PESU Academy.

Reply:
sorry , i passed the session in the function while calling , forgot to recieve the session in function params.
i will change it